src/components/List/RowContent.js

```diff
@@ -325,15 +332,18 @@ const ListRowContent = ({
               {title}
             </MultilineText>
           </ContentColumn>
-          <ContentColumn key="secondary" medium={6}>
+          {/* this class name is for automation purposes please do not remove or modify the name */}
+          <ContentColumn key="secondary" medium={6} className="column__content">
             <MultilineText
               key="collapsed"
               responsiveSize={{ xSmall: "hecto", medium: "kilo" }}
               className={classnames({
                 subtitle: true,
                 "list-row--subtitle": true,
                 "subtitle--active": !isOpen,
-                "subtitle--hidden": isOpen
+                "subtitle--hidden": isOpen,
+                // this class name is for automation purposes please do not remove or modify the name
+                "subtitle--collapsed": true
```

Comment:
This `MultilineText` component is only collapsed when `isOpen` equals true

Reply:
I got the idea from a couple of line below there is a `"subtitle--expanded: true"` logic